templates/http-py/metadata/snippets/component.txt

```diff
@@ -1,9 +1,10 @@
-[[component]]
-id = "{{project-name | kebab_case}}"
-source = "{{ output-path }}/app.wasm"
-[component.trigger]
+[[trigger.http]]
 route = "{{http-path}}"
-[component.build]
-command = "spin py2wasm app -o app.wasm"
+component = "{{project-name | kebab_case}}"
+
+[component.{{project-name | kebab_case}}]
+source = "{{ output-path }}/app.wasm"
+[component.{{project-name | kebab_case}}.build]
+command = "componentize-py -w spin-http componentize -p . -p $(python3 -c 'import site; print(site.getsitepackages()[0])') app -o app.wasm"
 workdir = "{{ output-path }}"
-watch = ["app.py", "Pipfile"]
+watch = ["*.py"]
```

Comment:
Should requirements.txt be watched?

Reply:
I would assume not as it does not have a direct or indirect effect on the built component unless the source changes or the installed version of packages changes.